src/test/resources/checks/checkstyle/declarativeServicesDependencyInjectionTest/DeclarativeServicesUsage.java

```diff
@@ -0,0 +1,14 @@
+import org.osgi.framework.BundleContext;
+import org.osgi.service.log.LogService;
+
+public class DeclarativeServicesUsage {
+  public void setLog(LogService l) {
+    log = l;
```

Comment:
I was looking at this code being completely flabbergasted, glad it is the code used as input for the tests :-D